README.md

```diff
@@ -92,6 +115,38 @@ public partial class YourNode : Node
 }
 ```
 
+#### Parameters

```

Comment:
I wouldn't repeat the same paragraph for C#. You can add a note that you can use parameters (in the C# section above), but they are documented in the wrapper, so the code sample is unnecessary.

Reply:
That's fair, I chose to repeat it because people using only one or the other would not have to read the other one, but I can simplify it with no issues.